gptel.el

```diff
@@ -950,9 +966,9 @@ buffer created or switched to."
       (visual-line-mode 1))
      (t (funcall gptel-default-mode)))
     (unless gptel-mode (gptel-mode 1))
-    (if (bobp) (insert (or initial (gptel-prompt-string))))
-    (goto-char (point-max))
     (skip-chars-backward "\t\r\n")
+    (if (bobp) (insert (or initial (gptel-prompt-prefix-string))))
+    (goto-char (point-max))
```

Comment:
The `(goto-char (point-max))` should come first, before the `(skip-chars-backward...`.